docs/index.md

```diff
@@ -0,0 +1,23 @@
+# Beetbox - Drupal development on Vagrant [![Circle CI](https://circleci.com/gh/drupalmel/beetbox.svg?style=svg)](https://circleci.com/gh/drupalmel/beetbox)
+
+## Overview
+
+Beetbox is a Vagrant configuration that provides your team with a versatile development environment for Drupal projects using Virtualbox for VMs (virtual machines).
+
+## Why use Beetbox?
+
+There are a number of Drupal-focussed Vagrant projects.
+
+* It is leaner than most other projects.
```

Comment:
I would say if anything that this whole file would likely be entirely re-written, as it needs to include the quickstart information... however there's no harm improving it either.

Feel free to make a PR to my fork (ensuring that if you change this file you also change README.md) or alternatively make a PR after I have accepted this one.

I would suggest that a statement like **Allows granular configuration** might need some explanation to it though.